site/topics/control-logic/instructions-microcodes.rst

```diff
@@ -18,6 +18,74 @@ Instructions and Microcodes
 Constraints
 ===========
 
+* The designers of the computer (us) decide what instructions to include and how they are to be performed
+* However, it is important to consider the limitations and constraints of the system
+
+* Although the ESAP system is to be Turing Complete, it is limited by the fact that it's an 8 bit system
+
+    * RAM is byte addressable
+    * The main bus is 8 bits wide
+
+
+* Thus, the information on the bus can only be 8 bits
+
+    * This information includes data
+    * But also includes *instructions*
+
+
+* Details on what the instructions are and how the system deals with them will be covered later
+* For now, consider that, for the ESAP system, individual instructions for the system will be made up of two parts
+
+    * Operation/operator
```

Comment:
imo pick either operation or operator and stick to it. Text is now _operation/operator_ in a few places, but only _operator_ in others.